xen/arch/arm/irq.c

```diff
@@ -30,11 +30,16 @@
 static unsigned int local_irqs_type[NR_LOCAL_IRQS];
 static DEFINE_SPINLOCK(local_irqs_type_lock);
 
+const int gsx_irq_num = 151;
+
+#define GSX_GUESTS_CNT    7
```

Comment:
I would have a comment here that 8th context is always used by the host driver, so this is why 7 is here

Reply:
will add a comment